Index: src/osapi/auth/osapiOAuth2Legged.php
===================================================================
--- src/osapi/auth/osapiOAuth2Legged.php	(revision 212)
+++ src/osapi/auth/osapiOAuth2Legged.php	(working copy)
@@ -88,7 +88,7 @@
    * @return string the signed url
    */
   public function sign($method, $url, $params = array(), $postBody = false, &$headers = array()) {
-    $oauthRequest = OAuthRequest::from_request($method, $url, $params);
+    $oauthRequest = new OAuthRequest($method, $url, $params);
 
     $params = $this->mergeParameters($params);
 
